compiler/passes/scopeResolve.cpp

```diff
@@ -880,15 +880,27 @@ static Expr* handleUnstableClassType(SymExpr* se) {
           if (DefExpr* d = toDefExpr(cur))
             inDef = d;
         }
-        // Find outer call, not counting baseExpr
+        // Find outer call, but don't count:
+        //  * baseExpr
+        //  * calls to types
+        //  * not counting baseExpr
```

Comment:
This comment is a little confusing for the last bullet. My brain parses it as "but don't count not counting baseExpr".